### PR TITLE
[GTD][Code Sample] React SPA Code Sample Update (ADO-54223)

### PR DESCRIPTION
**Pull request by OwenRichards1**
<html>
<body>

ID | Title | Assigned To | Iteration Path
-- | -- | -- | --
54223| [GTD][Code Sample] React SPA Code Sample Update | Owen Richards | Content\Production\Security Compliance and Identity\Microsoft Identity\GTD

@henrymbuguakiarie please verify this is correct, pretty sure I have done this correctly.
_____________________________________________________________________________________________________________________________________
@cilwerner @Dickson-Mwendia

### CODE REVIEW

**Comment by OwenRichards1** [comment]
#assign-reviewer: @henrymbuguakiarie

File: src/index.js [hunk 1]
```diff
@@ -1,104 +1,32 @@
 import React, { useState } from 'react';
```
Comment (henrymbuguakiarie) [inline src/index.js]: We import {useState} but it's never used.

File: src/index.js [hunk 2]
```diff
@@ -1,104 +1,32 @@
...
 import ReactDOM from 'react-dom';
+import './styles/index.css';
+import App from './App';
 
 // <ms_docref_import_modules>
-import { Configuration,  PublicClientApplication } from "@azure/msal-browser";
-import { MsalProvider, useMsal, useIsAuthenticated } from "@azure/msal-react";
-// </ms_docref_import_modules>
+import { Configuration, PublicClientApplication } from '@azure/msal-browser';
```
Comment (henrymbuguakiarie) [inline src/index.js]: We import {Configuration} but it's never used.

File: src/index.js [hunk 3]
```diff
@@ -1,104 +1,32 @@
...
+import { MsalProvider, useMsal, useIsAuthenticated  } from '@azure/msal-react';
```
Comment (henrymbuguakiarie) [inline src/index.js]: We import {useMsal, useIsAuthenticated} but we never use them in this file.

**Review by henrymbuguakiarie — COMMENTED** [review]
Hello @OwenRichards1, the codes works perfectly fine. I have point some instance where are importing context that we are not using. Otherswise, LGTM.

**Comment by OwenRichards1** [comment]
#sign-off

**Pull request closed without merging**